Source/ZoomNet/Resources/Groups.cs

```diff
@@ -1,6 +1,44 @@
+using Pathoschild.Http.Client;
+using System.Text.Json.Nodes;
+using System.Threading;
+using System.Threading.Tasks;
+
 namespace ZoomNet.Resources;
 
-public class Groups
+/// <summary>
+/// Allows you to manage Groups.
+/// </summary>
+public class Groups : IGroups
 {
-	
+	private readonly IClient _client;
+
+	internal Groups(IClient client)
+	{
+		_client = client;
+	}
+
+	/// <summary>
+	/// Adds a user to a group.
+	/// </summary>
+	/// <param name="groupId">The ID of the group.</param>
+	/// <param name="userEmail">The email of the user to add to the group.</param>
+	/// <param name="cancellationToken">The cancellation token.</param>
+	/// <returns>A task representing the operation. The result will be true if the user was added successfully, false otherwise.</returns>
+	/// <inheritdoc/>
+	public async Task<bool> AddUserToGroupAsync(string groupId, string userEmail, CancellationToken cancellationToken = default)
+	{
+		var data = new JsonObject
+		{
+			{ "members", new JsonArray(new JsonObject { { "email", userEmail } }) }
```

Comment:
To add multiple email addresses:

```csharp
{ "members", emails?.Select(e => new JsonObject { { "email", e } }) }
```

Reply:
This will not create the JSON structure we expect. Instead, it will create an incorrect representation because the value of "members" is an IEnumerable<JsonObject> instead of a `JsonArray`
You can take a look at the way I did it and let me know

Reply:
I have used the exact code I proposed elsewhere in the library and verified it with an integration test therefore I'm confident it works as expected. See [here](https://github.com/Jericho/ZoomNet/blob/develop/Source/ZoomNet/Resources/Chat.cs#L63).

Have you tested it? I'm curious what issue you observe in the resulting JSON.

Reply:
I used the code you provided, but encountered a serialization issue. Specifically, the` System.Text.Json` serializer threw an exception regarding missing type metadata for `IEnumerable<JsonObject>`.The serializer requires a more concrete type like `JsonArray `to handle the collection properly. Let me know if I am missing something.
 
In the example you provided, `AsObject<ChatChannel>()` is used, which could be employing specific deserialization settings that aren't used here, where we are trying to parse a general response (`AsJson()`).
 
 <details> 
 <summary>Serialization Exception Details</summary>
 System.NotSupportedException: JsonTypeInfo metadata for type 'System.Collections.Generic.IEnumerable`1[System.Text.Json.Nodes.JsonObject]' was not provided by TypeInfoResolver of type 'ZoomNet.Json.ZoomNetJsonSerializerContext'. If using source generation, ensure that all root types passed to the serializer have been annotated with 'JsonSerializableAttribute', along with any types that might be serialized polymorphically
 </details>

Reply:
Thanks for providing this additional information. I was able to reproduce the `'JsonTypeInfo metadata for type 'System.Collections.Generic.IEnumerable[System.Text.Json.Nodes.JsonObject]' was not provided` exception. I know that I have tested and used this code before so I'm surprised that it no longer works. My guess is that it used to work when we were using the Newtonsoft serializer and it stopped working when we transitioned to System.Text.Json but I am not 100% certain.

Anyway, long story short, the solution is rather simple: we simply need to change the content of the property from IEnumerable<JsonObject> to an array of JsonObject like so:
```csharp
{ "members", emails?.Select(e => new JsonObject { { "email", e } }).ToArray() } // <-- Notice .ToArray()
```

I will merge your PR and make the correction myself so we don't delay the release of this new feature. I will also raise an issue to correct the similarly faulty code in `Chat.CreateAccountChannelAsync`